source/opt/fold_spec_constant_op_and_composite_pass.cpp

```diff
@@ -160,26 +160,54 @@ bool FoldSpecConstantOpAndCompositePass::ProcessOpSpecConstantOp(
   return true;
 }
 
+uint32_t FoldSpecConstantOpAndCompositePass::GetTypeComponent(
+    uint32_t typeId, uint32_t element) const {
```

Comment:
Seems like a useful function for the type manager.

Reply:
I'll add an utility function to Instruction.